assets/queries/k8s/authorization_mode_set_to_always_allow/query.rego

```diff
@@ -0,0 +1,33 @@
+package Cx
+
+import data.generic.k8s as k8sLib
+
+CxPolicy[result] {
+	resource := input.document[i]
+	specInfo := k8sLib.getSpecInfo(resource)
+	container := specInfo.spec.containers[j]
+	commands := ["kube-apiserver", "kubelet"]
+
+	inArray(container.command, commands[_])
+	containerArgs := object.get(container, "args", {})
```

Comment:
```suggestion
```

I think this line does not make sense here since this rule will return a result if `--authorization-mode`  is set to `AlwaysAllow` in `container.args`. So, this rule considers that args is defined in the container

Additionally that, I think `--authorization-mod`e can be defined inside `container.command`.